renderers/src/main/java/com/pedrogomez/renderers/RVRendererAdapter.java

```diff
@@ -56,7 +61,15 @@ public T getItem(int position) {
     return position;
   }
 
-  /**
+  public void setCollection(AdapteeCollection<T> collection) {
+      if (collection == null) {
+          throw new IllegalArgumentException("Collection must be not null");
```

Comment:
The message shown will be more correct if we use something like ``The AdapteeCollection configured can't be null``.